src/goto-analyzer/static_simplifier.h

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************\
+
+Module:
+
+Author: Lucas Cordeiro, lucas.cordeiro@cs.ox.ac.uk
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_ANALYZER_STATIC_SIMPLIFIER_H
+#define CPROVER_GOTO_ANALYZER_STATIC_SIMPLIFIER_H
+
+#include <iosfwd>
+
+#include <util/message.h>
+#include <util/namespace.h>
```

Comment:
See above: this isn't need, and forward declarations are (personally) preferred.